pandas/tests/generic/test_finalize.py

```diff
@@ -197,12 +197,10 @@
     ),
     pytest.param(
         (pd.DataFrame, frame_data, operator.methodcaller("corr")),
-        marks=not_implemented_mark,
```

Comment:
Could you remove the outer `pytest.param` as well? (Same as below)

Reply:
Sure. The one below has another mark, which requires `pytest.param`.

I saw there are some `pytest.param` leftovers in the file. Should I also remove them, or just leave it for future maintenance?

Reply:
Got it. We can leave the one below then.

You can leave it for now and tackle it in a follow up PR if you'd like.